appveyor.yml

```diff
@@ -9,15 +9,16 @@ assembly_info:
   assembly_version: '{version}.0'
```

Comment:
Check out the `install` step in: https://github.com/serilog/serilog/blob/dev/appveyor.yml